tech/database/redis/about.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Redis
+subsection: redis
+section: tech-database
+description: In-memory data structure store, used as a database, cache and message broker
+---
+
+# Redis
+
+Redis is a free and open source in memory data structure store. It is widely used as a database, in memory cache or a message broker.
+
+## Quick start
+
+To install Redis on Fedora, run the following commands:
+
+```console
+$ sudo dnf install redis     # Install redis cli and server
+$ sudo systemctl start redis # Initialize redis server
+```
+
+To start redis on boot, run
+
+```console
+$ sudo systemctl enable redis
+```
+
+To test redis-cli, run
+
+```console
+$ redis-cli ping
+```
+
+the result from the above command will be 
```

Comment:
Please let's merge this into one block, with the above. Command output should be obvious (this is consistent with the rest of the website AFAIK).